lib/matplotlib/backends/backend_wxagg.py

```diff
@@ -1,10 +1,7 @@
 import wx
 
-import matplotlib
 from .backend_agg import FigureCanvasAgg
-from .backend_wx import (
-    _BackendWx, _FigureCanvasWxBase, FigureFrameWx,
-    NavigationToolbar2Wx as NavigationToolbar2WxAgg)
```

Comment:
That is kind of public API, i.e. some people may import the Toolbar classes from this backend module (yes, __all__ may be nice).

Reply:
Via [PEP8](https://www.python.org/dev/peps/pep-0008/#public-and-internal-interfaces):

> Imported names should always be considered an implementation detail. Other modules must not rely on indirect access to such imported names unless they are an explicitly documented part of the containing module's API

Other people "must not rely" on that working - if it becomes public API just because they might anyway, we might as well give up now!

Reply:
Maybe we should then add a comment that this is intended public  API even though the imports are not used within the module.

Reply:
My motto is still "small, incremental changes", so I've just taken out this change.  We can discuss what should happen here later; for now I'd rather get the rest of the PR merged!